pkg/service/service_test.go

```diff
@@ -1099,9 +1099,15 @@ func (m *ManagerTestSuite) TestUpsertServiceWithOnlyTerminatingBackends(c *C) {
 func (m *ManagerTestSuite) TestUpsertServiceWithExternalClusterIP(c *C) {
 	option.Config.NodePortAlg = option.NodePortAlgMaglev
 	option.Config.ExternalClusterIP = true
+	backends := make([]*lb.Backend, 0, len(backends1))
+	for _, b := range backends1 {
+		backends = append(backends, b.DeepCopy())
+	}
+	backends[0].State = lb.BackendStateActive
+	backends[1].State = lb.BackendStateActive
```

Comment:
Thanks for cleaning this up. Could you move the deep copy of backend structs and other similar changes to a separate commit, and add - 
Fixes: 22f5b525ebe ("service: Improve memory usage when handling update of a big service.") tag.

Reply:
There are more such changes that you forgot to move from the earlier commit - https://github.com/cilium/cilium/pull/24474/commits/54e6545a58fd967125692af59a6cf2973d0a984f.